assets/src/amp-story-editor-blocks.js

```diff
@@ -327,7 +327,12 @@ store.subscribe( () => {
 	}
 } );
 
-registerPlugin( 'amp-story-featured-image-pre-publish', { render: withPrePublishNotice } );
+registerPlugin(
+	'amp-story-featured-image-pre-publish',
+	{
+		render: StoryPrePublishPanel,
```

Comment:
Since the story actually won't display without the featured image, then I'm wondering if we should disable the `Publish` button completely if the featured image is not set? Thoughts?

Reply:
> Since the story actually won't display without the featured image

It actually will be displayed, thanks to #1998 and #2031. When there's absolutely no image whatsoever in a story, a fallback image is being used as the poster, allowing you to still view it.

But yeah we could experiment with disabling save in this case.

Reply:
Oh, right! Completely forgot about the featured image related updates 💡

Reply:
> Since the story actually won't display without the featured image, then I'm wondering if we should disable the `Publish` button completely if the featured image is not set? Thoughts?

Added a new card for this: https://github.com/ampproject/amp-wp/projects/10#card-19655884